README.md

```diff
@@ -312,5 +316,6 @@ Here are a few links to other hardware specific Node.js packages that may be of
 - [i2c-bus](https://github.com/fivdi/i2c-bus) - I2C serial bus access
 - [spi-device](https://github.com/fivdi/spi-device) - SPI serial bus access
 - [mcp-spi-adc](https://github.com/fivdi/mcp-spi-adc) - Analog to digital conversion with the MCP3002/4/8, MCP3202/4/8 and MCP3304
+- [pigpio-dht](https://github.com/depuits/pigpio-dht) - Implements logic to read DHT11 or DHT22/AM2302 temperature and relative humidity sensor
```

Comment:
Have you verified that pigpio-dht functions correctly?

Reply:
Amusingly enough I have a branch in a fork of it, https://github.com/erikma/pigpio-dht/tree/dev/erikma/fixRepeatedReadings, that fixes the 2nd and later readings. If all you want is one reading per node.js invocation it's OK, if like me you want to run readings every 2 minutes forever, you need my change which is what prompted me to issue this PR for pigpio.

Reply:
My intent for that fork is to first get a new pigpio npm version on npmjs, then update my fork and issue a PR against -dht using these changes.